### PR TITLE
Blogpost about iOS Appstore release

### PR DESCRIPTION
**Pull request by ghost**
@r10s Okay, the draft is ready so far :) Not entirely sure about planned features etc.

Became longer than I expected, but not too long I think :) 

Let's take a look at the preview, should appear at https://staging.delta.chat/240/ as soon as it is built.

### CODE REVIEW

**Comment by ghost** [comment]
I pinged you prematurely; I will fix the formatting and get back to you.

**Comment by ghost** [comment]
Alright @r10s I'm happy with it :) What do you think?

**Comment by hpk42** [comment]
i am skeptical of recommending ASMs for multi-device sync.  Better to recommend export + import even if more involved.  ASMs are designed for syncing with non-dc clients and only sync keys despite a lot of effort (typing in long numbers etc.).

**Comment by hpk42** [comment]
as @lefherz is likely not available due to travel let's modify the blog post with direct commits as needed. once 1.0.2 is published on app store the blog post can go live.  Thanks for kicking this off, @lefherz :)

File: _posts/2020-01-09-iOS-appstore-release.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<a href="https://apps.apple.com/app/delta-chat/id1459523234"
+target="_blank"><img src="../assets/badges/get-it-on-ios.png" width="200"
+style="margin: 10px;" /></a>
+
+<img src="../assets/blog/screenshots/2020-01-09-delta-chat-iOS-appstore-chat-list.jpg" width="250" style="float: right; margin: 10px;display: block;" /> 
+
+## What's Next?
+
+Some features are still missing in the iOS version - they will follow in the
+next months. Some examples are showing your profile picture to others, sharing
```
Comment (r10s) [inline _posts/2020-01-09-iOS-appstore-release.md]: "showing your profile picture to others" - not sure what is meant here. but ios has all profile picture functions android has - you can select a profile picture, it is sent to others and the profile pictures of others are shown as well.

File: _posts/2020-01-09-iOS-appstore-release.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Delta Chat Released for iPhones and iPads - Get it Now at the iOS Appstore!"
+author: compl4xx
+---
+
+<img src="../assets/blog/screenshots/2020-01-09-delta-chat-iOS-appstore-family-group-chat.jpg" width="200" style="float: right; margin: 10px;display: block;" />
```
Comment (r10s) [inline _posts/2020-01-09-iOS-appstore-release.md]: i will try to create a nicer screenshot including a phone borders. will do a commit here then.

a general hint: text from the first paragraph is shown in the blog-summary - however, if the first paragraph is only an image, nothing is shown there.

**Review by r10s — COMMENTED** [review]
great!

1.0.2 is on its way, once it is online, we can release the post :)

File: _posts/2020-01-09-iOS-appstore-release.md (hunk 2 above)
Comment (missytake) [reply _posts/2020-01-09-iOS-appstore-release.md]: Ah, putting the picture *into* the first paragraph is an idea I never thought of :) The wordpress-isation of blogpost writing took that away from me I guess :P

**Review by Simon-Laux — APPROVED** [review]

**Comment by Simon-Laux** [comment]
It's online so I merged it

**Comment by r10s** [comment]
thanks, @Simon-Laux

**Comment by hpk42** [comment]
well, this first sentence ... 

    "Until yesterday, you could only use Delta Chat on Android, Windows,
    MacOS, and Linux. Now you can have the Delta experience on all major
    platforms!"

... is factually wrong even though it sounds nice and "big news" dramatic.
I'd prefer to start off more like: 

    Finally, Delta Chat is available directly in Apple app store!  Thanks to the 
    many early Testflight users and to Bjoern, Richy and Basti doing all the heavy-lifting
    of developing Delta/iOS and making it to the app store ...

It's now too late for this correction/suggestion but let's avoid 
too "dramatical big news" stance. After all, the blog is read by the wider 
DC contributor and "interested followers" community and not so much from end-users. 
Most end-users will only read app store descriptions.  

There is thus no need to address the "general public" here but rather communicate 
well what's happening to contributors and followers.

**Comment by Simon-Laux** [comment]
> It's now too late for this correction/suggestion

You can still do it, I don't see a problem with it.

> let's avoid too "dramatical big news" stance

I agree

**Pull request merged**